packages/config/eslint.js

```diff
@@ -30,7 +30,7 @@ module.exports = {
   },
   rules: {
     '@typescript-eslint/ban-ts-comment': 'off',
-    '@typescript-eslint/no-explicit-any': 'off',
+    '@typescript-eslint/no-explicit-any': 'error',
```

Comment:
Also can you run `eslint` and fix all the places?

Reply:
yeah!